qa/t/rest_triage_owners.t

```diff
@@ -30,9 +30,6 @@ $t->get_ok(
   ->json_is('/Firefox/Installer/triage_owner', 'nobody@mozilla.org');
 
 my $data = $t->tx->res->json;
-use Bugzilla::Logging;
-use Mojo::Util qw(dumper);
-DEBUG(dumper $data);
```

Comment:
Unrelated?

Reply:
Yeah but harmless. I left it in there accidentally as part of a previous merge. I can make it a separate PR if you really want.

Reply:
No need, just thought I'd ask.